### PR TITLE
Improve CIELab docs

### PR DESCRIPTION
**Pull request by maxnoe**

### CODE REVIEW

**Comment by njsmith** [comment]
@KelSolaar Is L* unbounded too?

**Comment by KelSolaar** [comment]
Theoretically, it should be! CIE L* was developed for colorimetric measurements of coloured samples under a uniform illumination source however, as you know it, bounding it means data will be destroyed during conversions.

**Comment by njsmith** [comment]
@KelSolaar ah, good point! thanks

File: doc/reference.rst [hunk 1]
```diff
@@ -149,8 +149,9 @@ and 1.
 
 **CIELab**: The standard `CIE 1976 L*a*b* color space
 <https://en.wikipedia.org/wiki/Lab_color_space>`_. L* is scaled to
-vary from 0 to 100; a* and b* are likewise scaled to roughly the
-range -50 to 50. This space takes a parameter, *XYZ100_w*, which sets
+vary from 0 to 100; This color space is unbounded. 
+For colors inside the sRGB gamut, the values of a* and b* will be roughly in the range [-100, 100].
+This space takes a parameter, *XYZ100_w*, which sets
```
Comment (njsmith) [inline doc/reference.rst]: The semicolon + unrelated capitalized sentence is a bit odd. How about:

The standard CIE 1976 `L*a*b*` color space. `L*` is scaled to vary from 0 to 100, and in most settings the values of `a*` and `b*` will be roughly in the range -100 to 100, though you can certainly get larger values if your inputs are sufficiently extreme. This space takes a parameter...

**Comment by njsmith** [comment]
Thanks!

**Pull request merged**